sympy/geometry/line.py

```diff
@@ -2101,6 +2101,9 @@ def equation(self, x='x', y='y'):
         y : str, optional
             The name to use for the y-axis, default value is 'y'.
 
+        Note : Symbols being used need to be declared as real to avoid
```

Comment:
Should `Symbol`s always be declared as real? Or should they be declared with "appropriate" assumptions? If they always need to be real here, perhaps it can be done by the library rather than the user?

Reply:
I get what you are suggesting but I still have my doubts . It would make sense in this case if not declaring an assumption could be treated as declaring real ,it would be easier for the user if he isn't aware .